OvieMudi/Questioner__
# 163046612 implement all corrections by LFA Lanre

#### What does this PR do?
- Make html templates more responsive
- It changes the names of the signup and sign-in pages to follow the same convention
- Refactor code in the create and update routes of controllers
- Make sure single resource is duplicated during the creation
- Update tests files to reflect changes
_ Update README.md file
- _Correct failing test_
- _Username and email of user object cannot be changed via api_

#### Description of Task to be completed?
- Ensure the pages at https://oviemudi.github.io/Questioner__/UI/ are responsive
- Ensure single api returns data as an object instead of an array
- Ensure the POST api response at /api/v1/users returns an error response if duplicated (same username)
- Ensure the POST api response at /api/v1/meetups returns an error response if duplicated (same title)
#### How should this be manually tested?
- Open signin.html on the browser and ensure it loads correctly
- Tests specified enpoints at /api/v1/users and /api/v1/users via POSTMAN
- Test specified template at  https://oviemudi.github.io/Questioner__/UI/
#### Any background context you want to provide?
- This is a correction based on the issue of naming pointed out by Andela Bootcamp LFA Lanre
#### What are the relevant pivotal tracker stories?
[#163046612](https://www.pivotaltracker.com/story/show/163046612)
#### Screenshots
![pr-user cannot change email](https://user-images.githubusercontent.com/42499892/51040561-1ca12780-15b8-11e9-9918-97180e6ae480.PNG)
![pr-user cannot change username](https://user-images.githubusercontent.com/42499892/51040562-1d39be00-15b8-11e9-8819-040e4900305c.PNG)
![pr profile-page](https://user-images.githubusercontent.com/42499892/51268713-054fa900-19c1-11e9-8963-3fbefd5e044d.PNG)
